### ISSUE TITLE
can I use it for pytorch loss function?

### ISSUE THREAD

**Issue by zuujhyt** [Linear95/CLUB#5]
Hi,
```
output1 = model1(input)
output2 = model2(input)
```
I want to calculate the MI loss for ```output1``` and ```output2```
can I use your toolbox to achieve this purpose?
if yes, then can you please provide an example code snippet?
Thanks!

**Comment by Linear95** [comment]
Hello, I have uploaded a toy example of MI minimization with Gaussian samples in `mi_minimization.ipynb`. I think you can directly replace sample_x and sample_y with your output1 and output2 (I assume that your output1 has the shape [batch_size, embedding_dim]).

**Comment by zuujhyt** [comment]
Thanks for reply. I will try it.

**Comment by LindgeW** [comment]
How to use when output1 and output2 have the shape [batch_size, time_step, embedding_dim]??
